nipoppy/workflow/proc_pipe/tractoflow/run_tractoflow.py

```diff
@@ -595,7 +595,7 @@ def run(participant_id, global_configs, session_id, output_dir, use_bids_filter,
             tvec = bvec[:,tndir]
             tdir = np.unique(tvec, axis=0)
             mldir.append(tdir.shape[1])
-            
+
             ## compute and print the maximum lmax per shell
             tlmax = int(np.floor((-3 + np.sqrt(1 + 8 * tdir.shape[1]) / 2.0)))
```

Comment:
The formula requires more explanation / comments. If reused multiple times, then better to move it at the top.